src/pages/reactivistas/AddParticipantForm.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, FormEvent } from 'react';
+import { ReactGroup } from '../../lib/types';
+
+export const AddParticipantForm = ({ group }: { group: ReactGroup }) => {
+  const [discordUser, setDiscordUser] = useState('');
+  const [isSuccess, setIsSuccess] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+
+  const onAddParticipantSubmit = async (
+    event: FormEvent<HTMLFormElement>,
+    discordUser: string,
+    id: string,
+  ) => {
+    event.preventDefault();
+
+    const data = {
+      discordUser: discordUser,
+      id: id,
+    };
+
+    try {
+      setIsSuccess(true);
+      const res = await fetch('/api/add-participant', {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+    } catch (e) {
+      console.error(e);
+      setIsError(true);
+      setTimeout(() => setIsError(false), 5000);
+    }
+  };
+
+  return (
+    <form
+      onSubmit={(e) => {
+        onAddParticipantSubmit(e, discordUser, group._id);
+        setDiscordUser('');
+      }}
+      id={group.name}
+      className="flex"
+    >
+      <input
+        className="px-3 text-sm leading-tight text-gray-700 border rounded appearance-none focus:outline-none focus:shadow-outline"
+        name="discordUser"
+        type="text"
+        value={discordUser}
+        placeholder="Usuario de Discord"
+        required
+        disabled={isSuccess}
```

Comment:
Can we disable the button rather than the input? Also, I would make that button disabled as soon as it's clicked with an `isLoading` state.